src/main/kotlin/com/deviceinsight/helm/DeployMojo.kt

```diff
@@ -147,26 +150,28 @@ class DeployMojo : AbstractMojo() {
 
 		val chartTarGzFile = chartTarGzFile(chartDeploymentRequest)
 
-		val url = "${chartDeploymentRequest.chartRepoUrl}/api/charts"
+		val url = chartDeploymentRequest.chartRepoUrl
```

Comment:
This looks like a breaking change.
Although i don't like the hard-coded "/api/charts".

Reply:
Yes, indeed. I intentionally made it as breaking. 

What else we can do - introduce another 'chartLocation' which will contain /api/charts or have configuration in the plugin - artifactory/chartmuseum and then adjust urls based on it. What is your preference?

Reply:
Hm, yeah, I understand why you implemented it this way. The current, hard-coded implementation isn't optimal. But we use this plugin in a lot of projects and I'd like to avoid having all projects change their configuration.

I think I prefer the solution introducing a new property.